README.md

```diff
@@ -11,202 +11,273 @@ The goal of the project is to create a common HTML5 UI abstraction layer for the
+g++ build/basic.o build/webview.o $(pkg-config --libs gtk+-3.0 webkit2gtk-4.0) -o build/basic && build/basic
+# macOS
+g++ -c libs/webview/webview.cc -std=c++11 -o build/webview.o
+gcc -c basic.c -std=c99 -Ilibs/webview -o build/basic.o
+g++ build/basic.o build/webview.o -framework WebKit -o build/basic && build/basic
+# Windows/MinGW
+g++ -c libs/webview/webview.cc -std=c++17 -Ilibs/webview2/build/native/include -o build/webview.o
+gcc -c basic.c -std=c99 -Ilibs/webview -o build/basic.o
+g++ build/basic.o build/webview.o -Llibs/webview2/build/native/x64 -lWebView2Loader.dll -lole32 -lshell32 -lshlwapi -luser32 -o build/basic.exe && "build/basic.exe"
+```
+
+### Getting Started with Go
+
+See [Go package documentation][go-docs] for the Go API documentation, or simply read the source code.
+
+Install the package:
+
+```sh
+go mod init example.com/m
+go get github.com/webview/webview
 ```
 
-### C
-
-```c
-// main.c
-#include "webview.h"
-#include <stddef.h>
-
-#ifdef WIN32
-int WINAPI WinMain(HINSTANCE hInt, HINSTANCE hPrevInst, LPSTR lpCmdLine,
-                   int nCmdShow) {
-#else
-int main() {
-#endif
-	webview_t w = webview_create(0, NULL);
-	webview_set_title(w, "Webview Example");
-	webview_set_size(w, 480, 320, WEBVIEW_HINT_NONE);
-	webview_navigate(w, "https://en.m.wikipedia.org/wiki/Main_Page");
-	webview_run(w);
-	webview_destroy(w);
-	return 0;
-}
+On Windows you will need to make the WebView2 loader discoverable by cgo (see [Windows Preperation](#windows-preperation)).
+
+```bat
+set CGO_CPPFLAGS="-I%cd%\libs\webview2\build\native\include"
+set CGO_LDFLAGS="-L%cd%\libs\webview2\build\native\x64"
 ```
 
-Define C++ flags for the platform:
+Save the basic Go example into your project directory:
 
 ```sh
-# Linux
-$ CPPFLAGS="`pkg-config --cflags --libs gtk+-3.0 webkit2gtk-4.0` -lstdc++"
-# MacOS
-$ CPPFLAGS="-std=c++11 -framework WebKit"
-# Windows (x64) uses the build script
+curl -sSLo basic.go "https://raw.githubusercontent.com/webview/webview/master/examples/basic.go"
 ```
 
-Build it:
+Build and run the example:
 
 ```sh
-$ c++ -c $CPPFLAGS webview.cc -o webview.o  # build webview
-$ cc -c main.c -o main.o  # build C program
-$ c++ main.o webview.o $CPPFLAGS -o webview-example
+# Linux, macOS
+go build -o build/basic basic.go && ./build/basic
+# Windows
+go build -ldflags="-H windowsgui" -o build/basic.exe basic.go && "build/basic.exe"
 ```
 
-On Windows, run `script/build.bat`.
+> **Note:** On macOS you would typically [create a bundle](#macos-application-bundle) for your app with an icon and proper metadata for your app.
+
+### Try More Examples
+
+While the basic examples shown here may not be enough to get the whole picture, the good news is that the same procedure will work for all of our examples so please help yourself and try the other examples as well!
+
+We hope that you found it easy to get started using this library but if not then we have some more work to do and would appreciate [your feedback][issues-new].
```

Comment:
This section needs some simplification.

An idea:
```
## More Examples

These examples aren't enough to get the whole picture. Thankfully, the same procedure will work for the [other examples][examples] as well. Feel free to try them and give us [your feedback][issues-new] if you find any issues.
```

Reply:
I've changed the wording.